content/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -100,18 +100,41 @@ You can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scen
 
 ### User migration
 
-`influxd upgrade` migrates existing 1.x users and their permissions.
-However, it *does not migrate administrative users*.
+`influxd upgrade` migrates existing 1.x users and their permissions **except** the following users:
 
-To review users with admin permissions, in the InfluxDB 1.x CLI, run `show users`.
-Any users labeled "admin" *will not* be migrated.
+- [1.x admin users](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/#admin-users)
+- [1.x non-admin users](/{{< latest "influxdb" "v1" >}}/administration/authentication_and_authorization/#non-admin-users)
+  that have not been granted any privileges
+
+{{< expand-wrapper >}}
+{{% expand "Review 1.x user privileges" %}}
+**To review 1.x users with admin privileges**, run the following against your InfluxDB 1.x instance:
+
+```sql
+SHOW USERS
+```
+
+**To review the specific privileges granted to each 1.x user**, run the following for each user in your InfluxDB 1.x instance:
+
+```sql
+SHOW GRANTS FOR "<username>"
+```
+{{% /expand %}}
+{{< /expand-wrapper >}}
```

Comment:
thoughts on adding: If no grants appear, the user labeled “admin” will not be migrated.

Reply:
👍🏻